docs/reference/roles_matrix.rst

```diff
@@ -0,0 +1,47 @@
+Roles Matrix
+============
+
+The ``Sonata\UserBundle\Form\Type\RolesMatrixType`` was built to show all roles in a matrix view.
```

Comment:
Please comply with the sf docs standards:

> [Each line should break approximately after the first word that crosses the 72nd character (so most lines end up being 72-78 characters);](http://symfony.com/doc/current/contributing/documentation/standards.html)

Reply:
How to do this is there a cs fixer?

Reply:
I dont think there is, or we dont have it.

Reply:
but anyway this only happens to this line, to linebreak ot before and we should be good to go.

Reply:
You have to manually break line, there is no fixer for docs

Reply:
In vim: `:set textwidth=72`, then `gqq`